069-shared-volume-mount.md

```diff
@@ -0,0 +1,108 @@
+    sftp-output-file-odf-sftp-0        Bound    pvc-c11a68fa-5865-4310-9194-624a3942b872   100Gi      RWO            rook-cephfs    20h
+```
+
+- Get a terminal to one of the kafka-connect pod and create a file at /mnt/shared as below
+```
+    $ kubectl exec -it odf-connect-cluster-connect-7fbd9589c-d4tx6 -- bash
+        [kafka@odf-connect-cluster-connect-7fbd9589c-d4tx6 kafka]$ cd /mnt/shared/
+        [kafka@odf-connect-cluster-connect-7fbd9589c-d4tx6 shared]$ ls -lah
+        total 0
+        drwxrwsr-x 2 root  1001  1 Aug 23 09:27 .
+        drwxr-xr-x 1 root  root 20 Aug 23 09:22 ..
+        -rw-r--r-- 1 kafka 1001  0 Aug 23 09:27 testFile
+```
+
+- Get a terminal to another one of the kafka-connect pod and ensure the created file is visible
+```
+    $ kubectl exec -it odf-connect-cluster-connect-7fbd9589c-d4tx6 -- bash
+        [kafka@odf-connect-cluster-connect-7fbd9589c-d4tx6 kafka]$ cd /mnt/shared/
+        [kafka@odf-connect-cluster-connect-7fbd9589c-d4tx6 shared]$ ls -lah
+        total 0
+        drwxrwsr-x 2 root  1001  1 Aug 23 09:27 .
+        drwxr-xr-x 1 root  root 20 Aug 23 09:22 ..
+        -rw-r--r-- 1 kafka 1001  0 Aug 23 09:27 testFile
+```
+
+## Affected/not affected projects
+
+The proposed enhancement to the Strimzi Kafka Operator will primarily affect projects that rely on distributed environments and require high availability and fault tolerance. These projects will benefit from the improved management of shared volume mounts across nodes. However, projects that operate in a single-node environment or do not rely on shared volume mounts for their operations will not be directly affected by this proposal. It’s important to note that while not all projects may be directly affected, the overall robustness and efficiency brought about by this feature can indirectly benefit the entire Strimzi organisation.
+
+## Compatibility
+
+The proposed enhancement to the Strimzi Kafka Operator has been designed with both future and backward compatibility in mind. For future compatibility, the feature is built to be flexible and adaptable to accommodate potential changes or additions to the storage and access modes. This ensures that the feature remains relevant and useful as the system evolves. For backward compatibility, the feature maintains the existing functionality and behavior of the operator when the new options are not specified. This means that existing projects can continue to operate without any changes, ensuring a smooth transition and minimizing disruption to current operations..
+
+## Rejected alternatives
+
+While formulating this proposal, several alternatives were considered but ultimately rejected for various reasons:
+
+- **Using Local Storage**: Initially, the idea of using local storage for each pod was considered. However, this was rejected because it would not allow data sharing across pods, which is a crucial requirement for ensuring high availability and fault tolerance in distributed systems.
```

Comment:
I think it is fine to _reject_ local storage. But I do not think shared persistent volumes are a good basis for distributed systems as you suggest as they cause many limitations as to how _distributed_ they can be.